superset/assets/visualizations/main.js

```diff
@@ -10,6 +10,7 @@ const vizMap = {
   cal_heatmap: require('./cal_heatmap.js'),
   compare: require('./nvd3_vis.js'),
   directed_force: require('./directed_force.js'),
+  chord: require('./chord.jsx'),
```

Comment:
simply adding the mapping from the `chord` visualization type to the appropriate javascript modules. It's possible here to have multiple visualization types target the same module (look at nvd3.js!)